apps/s_server.c

```diff
@@ -103,6 +103,8 @@ static int keymatexportlen = 20;
 
 static int async = 0;
 
+static int use_sendfile = 0;
```

Comment:
what is this used for?

Reply:
> 
> 
> what is this used for?

Set this variable when specifying -sendfile

Reply:
Ahh, you need to use this thing to pass the flag across the functions...